### ISSUE TITLE
 LaunchProcess: failed to execvp: /home/user/path/binaryninja/QtWebEngineProcess

### ISSUE THREAD

**Issue by MD-Levitan** [Vector35/binaryninja-api#2058]
After starting the program displays such errors endlessly. I'm not sure, but it looks like that this problem takes a lot of CPU (Without analyzing any files it loads 40% CPU) .
 
```
./binaryninja --debug --stderr-log
python: preparing to initialize
python: using path '/usr/lib/python38.zip:/usr/lib/python3.8:/usr/lib/python3.8/lib-dynload'
python: initialized
python: importing binaryninja
python: redirecting
Loaded python3 plugin 'llvm-deobfuscator'
LaunchProcess: failed to execvp:
/home/user/path/binaryninja/QtWebEngineProcess
LaunchProcess: failed to execvp:
/home/user/path/binaryninja/QtWebEngineProcess
LaunchProcess: failed to execvp:
/home/user/path/binaryninja/QtWebEngineProcess
```
- Os: Debain 10/ Mate
- Plugins: I removed all plugins.
- Install all needed libraries and reinstall libs with QT.

**Comment by psifertex** [comment]
Can you try some of the suggestions in https://docs.binary.ninja/guide/troubleshooting.html#platforms ? These types of bugs are usually GPU driver issues when running inside of a VM or with specific video chipsets. Specifically, the `QT_OPENGL=angle` and https://docs.binary.ninja/guide/troubleshooting.html#virtualbox-and-vmware suggestions.

**Comment by psifertex** [comment]
If you use the graphical unzip utility unfortunately it doesn't make some files executable.

Try re-extracting with the command-line `unzip` command instead, or use `chmod +x` to make all the files in the BN folder and BN/plugins folders executable.

**Comment by MD-Levitan** [comment]
At first time faced such problem with gui zip.
Thanks for help.

**Comment by jeffli678** [comment]
this is fixed in dev 2.2.2543. Now the distributed zip will be extracted properly using either cli or gui unzip utility
